sdcm/cluster_azure.py

```diff
@@ -45,6 +47,7 @@ def __init__(self, azure_instance: VmInstance,  # pylint: disable=too-many-argum
         name = f"{node_prefix}-{region}-{node_index}".lower()
         self.node_index = node_index
         self._instance = azure_instance
+        self.scheduled_events_document_incarnation = -1
```

Comment:
maybe `last_event_document_incarnation` ? since I can only this is some kind of timestamp, so it would be clearer why are we saving it, and what we expect out of it

Reply:
ok, good idea. I also added some comment to clarify it even further.